Инструкция по запуску

Напишите, пожалуйста, в `README.md` инструкцию по запуску программы.

Судя по содержимому `package.json`, нужно запускать `npm run server`. Но оно у меня вылетает:
```
D:\Mazdaywik\Documents\Преподавание\УТП 2020\utp2020-11-spreadsheet>npm run server

> unitables@1.0.0 server D:\Mazdaywik\Documents\Преподавание\УТП 2020\utp2020-11-spreadsheet
> node src/server/main/main.js

internal/modules/cjs/loader.js:1172
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: D:\Mazdaywik\Documents\Преподавание\УТП 2020\utp2020-11-spreadsheet\src\server\main\main.
js
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1172:13)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'ERR_REQUIRE_ESM'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! unitables@1.0.0 server: `node src/server/main/main.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the unitables@1.0.0 server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Mazdaywik\AppData\Roaming\npm-cache\_logs\2020-09-29T18_27_18_572Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -11,6 +11,14 @@
 - Дмитрий Жук - [@ZhukDima](https://github.com/ZhukDima)
 - Милана Кондакова - [@MKondakova](https://github.com/MKondakova)
 
+## Запуск проекта
+
+``` shell
+npm i
+npm run build
+npm run server
+```
+
 ## Прогресс
 Проект находится на ранней стадии разработки.
 

